service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/StandardDirectionsOrderIssuedEventHandlerTest.java

```diff
@@ -120,16 +206,49 @@ void shouldNotifyAllocatedJudgeOfIssuedStandardDirectionsOrderWhenNotificationEn
         given(standardDirectionOrderIssuedEmailContentProvider.buildNotificationParametersForAllocatedJudge(caseData))
             .willReturn(expectedParameters);
 
-        standardDirectionsOrderIssuedEventHandler.notifyAllocatedJudgeOfIssuedStandardDirectionsOrder(
+        given(featureToggleService.isSendNoticeOfProceedingsFromSdo())
+            .willReturn(false);
+
+        standardDirectionsOrderIssuedEventHandler.notifyAllocatedJudgeOfIssuedSDOandNoticeOfProceedings(
             new StandardDirectionsOrderIssuedEvent(caseData));
 
         verify(notificationService).sendEmail(
             STANDARD_DIRECTION_ORDER_ISSUED_JUDGE_TEMPLATE, ALLOCATED_JUDGE_EMAIL_ADDRESS, expectedParameters,
             "12345");
+
+        verify(notificationService, never()).sendEmail(
+            SDO_AND_NOP_ISSUED_JUDGE, ALLOCATED_JUDGE_EMAIL_ADDRESS, expectedParameters,
+            "12345");
     }
 
     @Test
-    void shouldNotNotifyAllocatedJudgeOfIssuedStandardDirectionsOrderWhenNotificationDisabled() {
+    void shouldNotifyAllocatedJudgeOfIssuedSDOAndNoticeOfProceedingsWhenNotificationEnabled() {
+        final AllocatedJudgeTemplateForSDO expectedParameters = getAllocatedJudgeSDOTemplateParameters();
+
+        final CaseData caseData = caseData();
+
+        given(featureToggleService.isAllocatedJudgeNotificationEnabled(SDO)).willReturn(true);
+
+        given(standardDirectionOrderIssuedEmailContentProvider.buildNotificationParametersForAllocatedJudge(caseData))
+            .willReturn(expectedParameters);
+
+        given(featureToggleService.isSendNoticeOfProceedingsFromSdo())
+            .willReturn(true);
+
+        standardDirectionsOrderIssuedEventHandler.notifyAllocatedJudgeOfIssuedSDOandNoticeOfProceedings(
+            new StandardDirectionsOrderIssuedEvent(caseData));
+
+        verify(notificationService).sendEmail(
+            SDO_AND_NOP_ISSUED_JUDGE, ALLOCATED_JUDGE_EMAIL_ADDRESS, expectedParameters,
```

Comment:
these could be spaced on new lines for consistency/readability, but not really bothered